src/main/java/ladder/domain/game/Name.java

```diff
@@ -0,0 +1,56 @@
+package ladder.domain.game;
+
+import java.util.Objects;
+
+/**
+ * Created By mand2 on 2020-12-03.
+ */
+public class Name {
+
+    private final String name ;
+
+    public static final int NAME_LEN = 5;
+    public static final String MESSAGE_NAME_NOT_NULL = "이름은 빈 칸이 될 수 없습니다.";
+    public static final String MESSAGE_NAME_LEN_FIVE = "이름은 5자 이내여야 합니다.";
```

Comment:
변수 선언 순서는 상수 다음 인스턴스 필드가 존재합니다 ㅎㅎ
```suggestion
    public static final int NAME_LEN = 5;
    public static final String MESSAGE_NAME_NOT_NULL = "이름은 빈 칸이 될 수 없습니다.";
    public static final String MESSAGE_NAME_LEN_FIVE = "이름은 5자 이내여야 합니다.";
    
    private final String name ;
```

Reply:
핳 이런 실수를..! 감사합니다 👍